### PR TITLE
User/muzikovam/update zweconf

### PR DESCRIPTION
**Pull request by muzikovam**
Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [x] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [ ] Bugfix 
- [ ] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes 

#### Changes proposed in this PR

- Updated ZWECONF workflow

#### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _workflows_ *

---
* Description: _ Updated ZWECONF workflow_ *

---
* Part: _name of customizable file involved_  *

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number:

### CODE REVIEW

**Comment by JoeNemo** [comment]
@1000TurquoisePogs, do you need to review any of this??

**Review by achmelo — APPROVED** [review]

File: workflows/files/ZWECONF.xml [hunk 1]
```diff
@@ -2057,22 +2024,20 @@ echo '  zaas:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
...
 echo '          zosmf:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '            jwtAutoconfiguration: "auto"' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '            serviceId: "zosmf"' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
+echo '            jwtAutoconfiguration: auto' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
+echo '            serviceId: ibmzosmf' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
 echo '        authorization:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
 echo '          endpoint:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
 echo '            enabled: false' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
-echo '          provider: ""' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
+echo '          provider: "native"' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
```
Comment (pavelzlatnik) [inline workflows/files/ZWECONF.xml:2037]: when on 2030 line is provider without "", are you sure there on line 2037 should be provider with ""?

**Review by pavelzlatnik — CHANGES_REQUESTED** [review]
see my comment to provider

File: workflows/files/ZWECONF.xml (hunk 1 above)
Comment (pavelzlatnik) [reply workflows/files/ZWECONF.xml:2037]: when there is value, "" should be removed (and later should be "" removed also from sample.yaml I guess, where is provider without value).

Comment (muzikovam) [reply workflows/files/ZWECONF.xml:2037]: Where is it without the quotation marks? What do you mean by "on line"?

Comment (muzikovam) [reply workflows/files/ZWECONF.xml:2037]: That's a different provider. I know what you mean but we are simply trying to have the same file as is zowe-example.yaml. But to be fair it does not really matter. Zwe program would handle it with or without the quotation marks and you can't base on the only fact that the end variable is called the same.

Comment (pavelzlatnik) [reply workflows/files/ZWECONF.xml:2037]: image
when there is set value (native), there should not be used quotation marks. And technically it is the same kind of variable

Comment (muzikovam) [reply workflows/files/ZWECONF.xml:2037]: Well, each string here is the same kind of variable🙃 But please, have this conversation with someome who is taking care of zowe-example.yaml and not with me who only cares about the output being the same😊

**Review by pavelzlatnik — APPROVED** [review]
ok

**Pull request merged**